docs/macros.rst

```diff
@@ -614,3 +620,29 @@ API
                 string (e.g. ``'red'``, ``'#ddd'`` or ``'(250, 250, 250)'``), default to use configuration ``BOOTSTRAP_ICON_COLOR`` (default value is ``None``).
     :param title: The title of the icon for accessibility support.
     :param desc: The description of the icon for accessibility support.
+
+
+render_links()
```

Comment:
This macro is not very useful since the user can use the `urlize` filter directly.

Reply:
Indeed, refactored so that urlize does all this. Was not aware of that. Thanks.